src/inmanta/agent/resourcepool.py

```diff
@@ -245,7 +245,7 @@ async def get(self, member_id: TPoolID) -> TPoolMember:
         """
         Returns a valid pool member for the given id
         """
-        LOGGER.debug("%s: requesting %s", self.my_name(), self.render_id(member_id))
+        LOGGER.log(LOG_LEVEL_TRACE, "%s: requesting %s", self.my_name(), self.render_id(member_id))
```

Comment:
What is the difference between this log level trace and the debug level? I'm unsure to see when to use one over the other.

Reply:
Trace is one level lower than debug. 

The reason I reduced this that for every call to an executor (and they happen A LOT), 
1. for a cache hit we get two log lines at debug level, extremely close together.  (requesting and found)
2. for a cache miss we also get 2 (requested and creating) but not one when it is done creating. 

So this log line is at trace, in order to debug locking issues, which should be extremely rare.